Index: Backend/dbFillers/TextFiller.py
===================================================================
diff --git a/Backend/dbFillers/TextFiller.py b/Backend/dbFillers/TextFiller.py
--- a/Backend/dbFillers/TextFiller.py	
+++ b/Backend/dbFillers/TextFiller.py	
@@ -7,7 +7,7 @@
 rolename='student'
 
 def randomUsers():
-    file = open('dbFillers/users.txt', 'w')
+    file = open('users.txt', 'w')
     for i in range(20):
         name = random.choice(names)
         surname = random.choice(surnames)
@@ -17,10 +17,14 @@
     file.close()
 
 def randomGuardian():
-    file = open('dbFillers/guardians.txt', 'w')
+    file = open('guardians.txt', 'w')
     for i in range(5):
         name = random.choice(names)
         surname = random.choice(surnames)
         email = name.lower() + '.' + surname.lower() + email_end
         file.write(name + " " + surname + " " + email +"\n")
-    file.close()
\ No newline at end of file
+    file.close()
+
+
+randomUsers()
+randomGuardian()
\ No newline at end of file
